controllers/discordactions.js

```diff
@@ -282,7 +282,7 @@ const updateDiscordNicknames = async (req, res) => {
         } catch (error) {
           logger.error(`error getting user with matching discordId ${error.message}`);
         }
-      })
+      }),
```

Comment:
same here

Reply:
These file changes are due to the linting fixes of new prettier package upgrade.

Reply:
![image](https://github.com/Real-Dev-Squad/website-backend/assets/71277980/b0cf4f5b-3cbe-4ada-8c04-b85c35005d73)